packages/components/src/internal/components/forms/QueryInfoForm.tsx

```diff
@@ -125,15 +115,6 @@ export class QueryInfoForm extends React.PureComponent<QueryInfoFormProps, State
         };
     }
 
-    UNSAFE_componentWillMount(): void {
-        const { schemaQuery } = this.props;
-        selectRows({
-            schemaName: schemaQuery.schemaName,
-            queryName: schemaQuery.queryName,
-            maxRows: 0,
-        });
```

Comment:
Odd, any ideas why this `selectRows` was here before when it didn't do anything with the response? I assume it was just never cleaned up / removed after deciding it wasn't necessary.

Reply:
I suspect this was being used to somehow interact with redux to load the associated `QueryInfo` as we may not have originally exposed a way to directly do that. I checked all of the usages to see if any were not providing a `queryInfo` but there were none.